br/pkg/lightning/restore/restore.go

```diff
@@ -2142,13 +2149,24 @@ type deliverResult struct {
 	err      error
 }
 
+func (cr *chunkRestore) adjustRowID(rowID *int64) {
```

Comment:
It's better to pass by value that passes an int64 value and returns an int64 value.